cli/src/commands/sidekick.rs

```diff
@@ -37,6 +38,14 @@
 		#[structopt(long)]
 		namespace: String,
 	},
+	GetNamespaceDevToken {
+		#[clap(short = 'n', long)]
```

Comment:
Do we need shorthand flags in the sidekick, since it's always called programatically?